.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: go
+
+go:
+ - 1.9
+ - tip
+
+matrix:
+  allow_failures:
+   - go: tip
+
+  fast_finish: true
+
+before-script:
+ - go get github.com/gorilla/mux
```

Comment:
I think we can combine these into a single `go get`, which will read through your source and detect the packages it needs to download